migration/secure_note_transformer.py

```diff
@@ -58,7 +58,7 @@ def _parse(self):
                 parsed_data[key] = value
             return parsed_data
         except:
-            logging.warning("Failed to parse LastPass data")
+            pass  # silence error here but prints in the vault_item_import.py
```

Comment:
I don't think I see any place in `vault_item_import.py` a place about being unable to parse LastPass data. The only moment where we skip some LastPass data is because we don't support that conversion yet.
Is it logged somewhere still? Or can this change be better explained?